### PR TITLE
feat: Create Latvian Dict

### LINKED ISSUE #2486: Latvian language support
Would be nice to have also Latvian language support

### PR DESCRIPTION
**Pull request by KieranRobson**
# Add/Fix Dictionary

Dictionary: lv

## Description

Create latvian Dict
Closes #2486

## References

https://www.101languages.net/latvian/most-common-latvian-words/ - Took the Complete List from this.
## Checklist

- [x] By submitting this issue, you agree to follow our
      [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)
- [x] Verify that the title starts with the correct prefix:
  - `fix:` - for minor changes like adding words or fixing spelling issues.
  - `feat:` - for a significant change like adding a whole new set of words to a dictionary.
  - `feat!:` - for breaking changes, like file format or licensing changes.
  - `chore:` - for changes that do not impact the content of dictionaries.

### CODE REVIEW

File: dictionaries/medicalterms/checksum.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-8438032358b0863d385a64143b75ece05cf261d7  medicalterms-en.txt.gz
+6053fc00ec7a37da3a5b73eb772a69fa32921c41  medicalterms-en.txt.gz
```
Comment (Jason3S) [inline dictionaries/medicalterms/checksum.txt:1]: @KieranRobson,

This checksum should not have changed, please make sure all the dependencies are up to date: `pnpm i` from the root.

**Comment by Jason3S** [comment]
@KieranRobson,

Thank you!

File: dictionaries/medicalterms/checksum.txt (hunk 1 above)
Comment (Jason3S) [reply dictionaries/medicalterms/checksum.txt:1]: Is it possible your fork is behind the main fork? There was a change in the tools that prevented the `.gz` files from being different if they were built on Linux, MacOS,or Windows.

Comment (KieranRobson) [reply dictionaries/medicalterms/checksum.txt:1]: I believe this was because I was behind on the main branch

**Pull request merged**